frontend/src/components/elements/PlotlyChart/CustomTheme.tsx

```diff
@@ -0,0 +1,471 @@
+  const { genericFonts, colors, fontSizes, fontWeights } = theme
+
+  const themeFonts = {
+    labelFont: genericFonts.bodyFont,
+    titleFont: genericFonts.bodyFont,
+    labelFontSize: fontSizes.twoSmPx,
+    titleFontSize: fontSizes.twoSmPx,
+  }
+
+  const streamlitTheme = {
+    background: colors.bgColor,
+    axis: {
+      labelColor: colors.bodyText,
+      titleColor: colors.bodyText,
+      gridColor: colors.fadedText10,
+      ...themeFonts,
+    },
+    header: {
+      labelColor: colors.bodyText,
+    },
+    view: {
+      continuousHeight: 350,
+      continuousWidth: 400,
+    },
+    font: {
+      family: genericFonts.bodyFont,
+    },
+    title: {
+      color: theme.colors.headingColor,
+      subtitleColor: colors.bodyText,
+      ...themeFonts,
+      font: {
+        family: genericFonts.bodyFont,
+        size: fontSizes.smPx + 2,
+        color: colors.headingColor,
+      },
+      // text: (layout.title.text) ? `<b>${layout.title.text}</b>` : undefined,
+      x: 0,
+    },
+    colorway: hasLightBackgroundColor(theme)
+      ? [
+          "#0068C9",
+          "#83C9FF",
+          "#FF2B2B",
+          "#FFABAB",
+          "#29B09D",
+          "#7DEFA1",
+          "#FF8700",
+          "#FFD16A",
+          "#6D3FC0",
+          "#D5DAE5",
+        ]
+      : [
+          "#83C9FF",
+          "#0068C9",
+          "#FFABAB",
+          "#FF2B2B",
+          "#7DEFA1",
+          "#29B09D",
+          "#FFD16A",
+          "#FF8700",
+          "#6D3FC0",
+          "#D5DAE5",
+        ],
+    legend: {
+      // side: "left",
+      // orientation: "h",
+      // yanchor: "bottom",
+      // xanchor: "left",
+      // y: -.25,
+      title: {
+        font: {
+          size: fontSizes.smPx,
+          color: getGreyText(theme),
+        },
+        side: "top",
+      },
+      valign: "top",
+      bordercolor: colors.transparent,
+      borderwidth: 0,
+      // tracegroupgap: 200,
+      font: {
+        color: getGreyText(theme),
+      },
+      labelColor: getGreyText(theme),
+      titleColor: getGreyText(theme),
+      ...themeFonts,
+    },
+    paper_bgcolor: colors.bgColor,
+    plot_bgcolor: colors.bgColor,
+    // yaxis: {
+    //   minor: {
+    //     showgrid: false,
+    //   }
+    // },
+    colorDiscreteSequence: hasLightBackgroundColor(theme)
+      ? [
+          "#0068C9",
+          "#83C9FF",
+          "#FF2B2B",
+          "#FFABAB",
+          "#29B09D",
+          "#7DEFA1",
+          "#FF8700",
+          "#FFD16A",
+          "#6D3FC0",
+          "#D5DAE5",
+        ]
+      : [
+          "#83C9FF",
+          "#0068C9",
+          "#FFABAB",
+          "#FF2B2B",
+          "#7DEFA1",
+          "#29B09D",
+          "#FFD16A",
+          "#FF8700",
+          "#6D3FC0",
+          "#D5DAE5",
+        ],
+    yaxis: {
+      // nticks: 10,
+      ticklabelposition: "outside",
+      tickmode: "auto",
+      // automargin: "left",
+      zeroline: false,
+      title: {
```

Comment:
You can add `standoff` here to add some additional spacing between the title and the axis:
https://plotly.com/python/reference/layout/yaxis/#layout-yaxis-title-standoff